packages/yew/src/dom_bundle/app_handle.rs

```diff
@@ -1,17 +1,16 @@
-//! This module contains the `App` struct, which is used to bootstrap
```

Comment:
How is this module related to `dom_bundle`?

Reply:
`dom_bundle` contains state to keep when mounting to the DOM. I think it's just natural that the `AppHandle` is here, since it contains the state for a whole app - concretely the parent scope.